function test(){
    try{

    }catch(err) {
        if(x){
            return 0;
        }
        
    }finally{
        return 0;
    }
}